src/components/Admin/index.jsx

```diff
@@ -232,7 +231,7 @@ class Admin extends React.Component {
 
     return (
       <React.Fragment>
-        {!loading && !error && !this.hasAnalyticsData() ? <NotFoundPage /> : (
+        {!loading && !error && !this.hasAnalyticsData() ? this.renderLoadingMessage() : (
```

Comment:
This probably fixes it, but I'm confused why `loading` would be false if we're currently loading (and showing the loading message)

Reply:
it's confusing because one is the loading state for the page and the other is the "loading" state just for analytics data. I'd have to look more into the code to consolidate them, but idea is that since that function is waiting on data, it will return false for that time -- thus triggering the 404. Since its _really_ not a missing page (its just awaiting data), I figured loading here a second time was a workable solution.